web/src/narrow.js

```diff
@@ -125,14 +124,14 @@ export function activate(raw_terms, opts) {
         raw_terms = [{operator: "is", operand: "home"}];
     }
     const filter = new Filter(raw_terms);
+    const filter_to_all_messages_view = filter.is_in_home();
```

Comment:
`is_combined_feed_global_view` might be a good name for variable, given the current naming plans.

Reply:
Changed.